scripts/quality/check_ownership.py

```
@@ -0,0 +1,78 @@
+        sys.exit(1)
+
+    with open(ownership_path) as f:
+        return yaml.safe_load(f)
+
+
+def get_changed_files():
+    import subprocess
+
+    try:
+        result = subprocess.run(
+            ["git", "diff", "--name-only", "origin/main...HEAD"],
```

Ruff (0.15.1)

[error] Starting a process with a partial executable path

(S607)